tw-tasks-management/src/main/java/com/transferwise/tasks/management/ITasksManagementService.java

```diff
@@ -207,4 +213,19 @@ public static class Task {
       private Instant stateTime;
     }
   }
+
+  GetTaskTypesResponse getTaskTypes();
+
+  @Data
+  @Accessors(chain = true)
+  class GetTaskTypesResponse {
+    private List<TaskType> types;
+
+    @Data
+    @Accessors(chain = true)
+    public static class TaskType {
+      private String type;
+      private List<String> subTypes;
```

Comment:
A task type can have a list of Subtypes?

Later in the same PR, it looks like you're only `setting` one subtype?